tachyon/zk/plonk/circuit/phase.h

```diff
@@ -34,8 +34,8 @@ class TACHYON_EXPORT Phase {
 
   uint8_t value() const { return value_; }
 
-  bool operator==(const Phase& other) const { return value_ == other.value_; }
-  bool operator!=(const Phase& other) const { return value_ != other.value_; }
+  bool operator==(Phase other) const { return value_ == other.value_; }
```

Comment:
Can you add a note that passing value here is on purpose?

Reply:
It's already specified at the comment above the class! Otherwise, you should put these comments everywhere!